src/components/browse/datasetlist/datasetList.tsx

```diff
@@ -55,14 +58,24 @@ const DatasetList = (props: dataSetProps) => {
   };
 
   const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
-    console.log(event)
-    props.setLimit(parseInt(event.target.value))
+    console.log(event);
```

Comment:
please remove console.log methods

Reply:
Interesting. I guess we didn't remove it for a while. Removed on latest commit.